src/modules/ExactTracking.jl

```diff
@@ -7,8 +7,8 @@
 struct Exact end
 
 module ExactTracking
-using ..GTPSA, ..BeamTracking, ..StaticArrays, ..ReferenceFrameRotations, ..KernelAbstractions
-using ..BeamTracking: XI, PXI, YI, PYI, ZI, PZI, Q0, QX, QY, QZ, @makekernel, Coords
+using ..GTPSA, ..BeamTracking, ..StaticArrays, ..ReferenceFrameRotations, ..KernelAbstractions, ..SIMD, ..SIMDMathFunctions
+using ..BeamTracking: XI, PXI, YI, PYI, ZI, PZI, Q0, QX, QY, QZ, State_Alive, State_Lost, @makekernel, Coords
```

Comment:
```suggestion
using ..BeamTracking: XI, PXI, YI, PYI, ZI, PZI, Q0, QX, QY, QZ, STATE_ALIVE, STATE_LOST, @makekernel, Coords
```
Constants should be all caps. The rest of the `State_alive` `State_lost` in this file should be changed to all caps